<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmMpmGui</class>
 <widget class="QMainWindow" name="EmMpmGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>784</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EM/MPM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="zoomIn">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/icons/images/Icons.qrc">
            <normaloff>:/zoomIn.png</normaloff>:/zoomIn.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="zoomOut">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/icons/images/Icons.qrc">
            <normaloff>:/zoomOut.png</normaloff>:/zoomOut.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fitToWindow">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/icons/images/Icons.qrc">
            <normaloff>:/fitToWindow.png</normaloff>:/fitToWindow.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="zoomCB">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>10%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>125%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>600%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fit to Window</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addUserInitArea">
        <property name="text">
         <string>+ Area</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Composition Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="compositeModeCB"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="EMMPMGraphicsView" name="m_GraphicsView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_FixedRecentFiles">
     <property name="title">
      <string>Fixed Recent Files</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menu_MovingRecentFiles">
     <property name="title">
      <string>Moving Recent Files</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionOpenFixedImage"/>
    <addaction name="actionOpenMovingImage"/>
    <addaction name="menu_FixedRecentFiles"/>
    <addaction name="menu_MovingRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Canvas"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFixedImage">
   <property name="text">
    <string>Open Fixed Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionSave_Canvas">
   <property name="text">
    <string>Save Canvas...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenMovingImage">
   <property name="text">
    <string>Open Moving Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EMMPMGraphicsView</class>
   <extends>QWidget</extends>
   <header location="global">EMMPMGraphicsView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
